test/language/class/class-name-binding.js

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) Copyright 2014 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 14.5
+description: >
+    class name binding
+---*/
+var C2;
+class C {
```

Comment:
Gah, [there's already a test for that](https://github.com/bocoup/test262/blob/harmony-classes/test/language/class/class-name-binding-expression.js). Sorry for the noise; I'll try to be a little more patient!